terraform/templates/automate-config.toml

```diff
@@ -0,0 +1,107 @@
+# This is a default Chef Automate configuration file. You can run
+# 'chef-automate deploy' with this config file and it should
+# successfully create a new Chef Automate instance with default settings.
+
+[global.v1]
+  # The external fully qualified domain name.
+  # When the application is deployed you should be able to access 'https://<fqdn>/'
+  # to login.
+  fqdn = "${automate_fqdn}"
+
+  # TLS Certificates for External-Facing Services
+    # The following TLS certificate and RSA public key were
+    # automatically generated. The certificate is a self-signed
+    # certificate and will likely produce security warnings when you
+    # visit Chef Automate in your web browser. We recommend using a
+    # certificate signed by a certificate authority you trust.
+  [[global.v1.frontend_tls]]
+    # The TLS certificate for the load balancer frontend.
+    cert = """-----BEGIN CERTIFICATE-----
```

Comment:
Can we automate the cert generation using lets-encrypt or something?

Reply:
That's future work.  I think the best way to do this would be to leverage Route53 + AWS Certificate Manager.   That way in a single shot we:
1. Create a DNS record
2. Request an ACM cert for that record, or use a wildcard cert (that's what I'm doing now with *.success.chef.co )
3. apply the cert to the ELB - that's where SSL will be terminated